### PR TITLE
logo on gh pages

### PR DESCRIPTION
**Pull request by entonbiba**
update github docs website with new logo, 

- [x]  upload logo
- [x]  update css
- [x] update html

related pr #1420 

image
image

### CODE REVIEW

**Comment by katspaugh** [comment]
Awesome! Could you make the homepage logo aligned with the left side, like the previous one?

**Comment by entonbiba** [comment]
@katspaugh done,

File: css/style.css [hunk 1]
```diff
@@ -25,9 +25,14 @@ header {
     padding-left: .9375rem;
 }
 
-header h1 {
-    font-family: 'Knewave', cursive;
```
Comment (katspaugh) [inline css/style.css]: Btw, I guess we can now delete the [Knewave import](https://github.com/katspaugh/wavesurfer.js/blob/gh-pages/css/style.css#L1), too.

Comment (thijstriemstra) [reply css/style.css]: done.

**Pull request merged**